toolkits/landcover/test/unit/landsat8.unit.test.js

```diff
@@ -30,4 +30,9 @@ withEarthEngineStub('Landsat8', function() {
     var expected = originalImage.updateMask(mask);
     expect(newImage).toEqual(expected);
   });
+
+  it('getTasseledCapCoefficients() returns coefficients.', function() {
```

Comment:
This coefficients are a constant, so this test isn't needed. We really only want to unit test business logic, which in our case happens to mostly consist of configuring a graph of `ee` objects.